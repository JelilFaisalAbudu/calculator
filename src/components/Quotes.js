/* eslint-disable max-len */
import React from 'react';
import quotesStyles from '../styles/quotes.module.css';

function Quotes() {
  return (
    <div className={quotesStyles['quote-container']}>
      <p className={quotesStyles.quote}>
        Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. â€“William Paul Thurston
      </p>
    </div>
  );
}

export default Quotes;
