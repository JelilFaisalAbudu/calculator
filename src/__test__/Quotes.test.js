import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Quotes from '../components/Quotes';

describe('Quotes Component', () => {
  test('renders part text of the Quotes page', () => {
    const { getByText } = render(<Quotes />);
    const text = getByText(/Mathematics is not about numbers,/);

    expect(text).toBeInTheDocument();
    expect(text).toMatchSnapshot();
  });
});
