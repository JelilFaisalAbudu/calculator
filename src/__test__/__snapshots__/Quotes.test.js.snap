// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quotes Component renders part text of the Quotes page 1`] = `
<p
  class="quote"
>
  Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. â€“William Paul Thurston
</p>
`;
